docs/make.jl

```diff
@@ -62,3 +64,25 @@ makedocs(
     ],
     draft = false,
 )
+
+"""
+    recursive_find(directory, pattern)
+
+Return list of filepaths within `directory` that contains the `pattern::Regex`.
+"""
+function recursive_find(directory, pattern)
+    mapreduce(vcat, walkdir(directory)) do (root, dirs, filenames)
+        matched_filenames = filter(contains(pattern), filenames)
+        map(filename -> joinpath(root, filename), matched_filenames)
+    end
+end
+
+@info "Cleaning up temporary .jld2 and .nc output created by doctests or literated examples..."
```

Comment:
Note that we're already cleaning up jld2 files before deploying the website (see #136), but I guess it's good to remove them after the build anyway if they aren't necessary (but perhaps they might be useful if someone wants to quickly do a new plot locally without rerunning the simulation?)

Reply:
Yes, the GH action removes the outputs from literate examples. But sometimes also doctests produce output (e.g. Oceananigans output writers doctests do that) so this was my way to cover all bases. If you have a cleaner suggestion it'd be great!